Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt

```diff
@@ -119,7 +119,7 @@ class PersonDetailsFragmentTest : BaseUITest() {
                 listOf(testCertificate, vaccinationCertificate1, vaccinationCertificate2), isCwaUser = isCwa
             )
 
-            add(PersonDetailsQrCard.Item(testCertificate, bitmap))
+            add(PersonDetailsQrCard.Item(testCertificate, bitmap) {})
```

Comment:
Should this `{}` do something later on? Maybe add a TODO?

Reply:
it is a test , won't do anything